docs/editors-guide/curation-workflow-overview.md

```diff
@@ -0,0 +1,63 @@
+# Mondo Curation Workflow Overview
+
+## Description
+This document describes the community curation workflow in Mondo, including how new terms are added to Mondo, when terms are obsoleted, and Mondo releases. 
+
+## Adding new terms to Mondo
+- New terms are added to Mondo via two processes:
+  1. User requests via our [GitHub tracker](https://github.com/monarch-initiative/mondo/issues).
+  2. 'Slurping' from [external sources](https://mondo.monarchinitiative.org/pages/sources/) - this is our pipeline to ingest new terms from external sources (such as OMIM, Orphanet), to keep Mondo synched with the external sources.
+  
```

Comment:
note that once I am done with issue 5848, we will have a 3rd way to submit terms

Reply:
yeah, we can add that when #5848 is done